src/patternfly/components/Button/styles.scss

```diff
@@ -69,29 +70,29 @@
   // Tertiary btn
   --pf-c-button--m-tertiary--BackgroundColor: transparent;
   --pf-c-button--m-tertiary--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-button--m-tertiary--Color: var(--pf-color-black-900);
+  --pf-c-button--m-tertiary--Color: var(--pf-global-Color--100);
```

Comment:
whomp whomp still missing one between `global` and `Color` `--pf-global--Color--100` Wonder if there is a utility to verify these.  I might take a look to see if I can find one.